.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+<!-- Thanks for contributing to _vlc-ios_! Before you submit your pull request, please make sure to check the following boxes by putting an x in the [ ] (don't: [x ], [ x], do: [x]) -->
+
+### Checklist
+- [ ] I've run `bundle exec fastlane test` from the root directory to see all new and existing tests pass
+- [ ] I've followed the vlc-ios code style and run `bundle exec fastlane lint` to ensure the code style is valid
```

Comment:
can we adjust the code style.rtf (which probably should turn into an .md file) and link to it here ? :)

Reply:
I could not agree more... done ✅